src/QmlControls/ScreenTools.qml

```diff
@@ -107,7 +108,10 @@ Item {
         property real   fontWidth:    contentWidth
         property real   fontHeight:   contentHeight
         Component.onCompleted: {
-            var baseSize = QGroundControl.baseFontPointSize;
+            var baseSize = QGroundControl.corePlugin.options.defaultFontPointSize
```

Comment:
It needs to define a baseline default. I have to rethink this whole thing at some point. Once I do that, we can go through these and make them right.